documentation/testing/test-plans/20240711_release-test-plan.md

```diff
@@ -0,0 +1,81 @@
+
+- [216189- Update Cosmos connection details for Web App to use Managed Identity](https://dfe-ssp.visualstudio.com/s198-DfE-Benchmarking-service/_workitems/edit/216189)
+
+This update has impacted the login side of the service so all areas which requires login custom data, comparators, school details, CFP journey has been tested as part of this. Also sanity checks are performed on the whole service to ensure the service is working as expected.
+
+- [216190 - Update SQL DB connection details for APIs to use Managed Identity](https://dfe-ssp.visualstudio.com/s198-DfE-Benchmarking-service/_workitems/edit/216190)
+
+This has impacted the connection details for APIs so checked school/trusts/LAs areas where data is pulled which includes comparison page, historic data ,benchmarking page, spending and costs and other pages and validated that the data is still displayed on the pages which ensured the connection is working as expected. 
+
+- [218281 -Incorrect responses from authorization attributes](https://dfe-ssp.visualstudio.com/s198-DfE-Benchmarking-service/_workitems/edit/218281)
+
+This impacted the response code when the user is trying to access schools which user don't have permission. verified the response code sent back as 401 and now been updated to 403 (server understand the request but refuse to authorise it)
+
+- [217961 - Graceful error handling on proxy API end points](https://dfe-ssp.visualstudio.com/s198-DfE-Benchmarking-service/_workitems/edit/217961)
+
+if the school don't have current year data then benchmarking pages are throwing 500 error which impacts the analytics. The error has been handled gracefully now. 
+
+- [218339 -Trusts are not showing in search bar](https://dfe-ssp.visualstudio.com/s198-DfE-Benchmarking-service/_workitems/edit/218339)
+
+This is related to Raw data that was used for trust details. Updated it to read Trust details from GIAS files which has resolved this issue. We have validated this by checking existing trust names with the updated ones and the ones that are not matching were compared with GIAS files. This ensures now we have correct trusts as expected. 
+
+- [218046 - Learn@ MAT appears to be using incorrect Group UID](https://dfe-ssp.visualstudio.com/s198-DfE-Benchmarking-service/_workitems/edit/218046)
+
+This is related to the raw data that was used for trust details which was causing duplicate UIDs. We have ensured now that there are no duplicate UIDs for the trusts which are linked with schools. 
+### Issues Not to be Tested:
+n/a
+
+## Test Deliverables
+### Documents:
+Summary of testing performed and outcomes. 
+
+### Reports:
+Release test plan
+
+## Entry and Exit Criteria
+### Entry Criteria:
+- all changes are in preprod after passing through the earlier quality gates 
+
+### Exit Criteria: 
+- testing is complete
+- logged issues are fixed and retested
```

Comment:
Is this current? Would be fix **all** logged issues?

Reply:
updated it now.